go/ql/test/library-tests/semmle/go/frameworks/Protobuf/TaintFlows.ql

```diff
@@ -1,27 +1,18 @@
 import go
+import TestUtilities.InlineFlowTest
 
-class UntrustedFunction extends Function {
-  UntrustedFunction() { this.getName() = ["getUntrustedString", "getUntrustedBytes"] }
-}
-
-class UntrustedSource extends DataFlow::Node, UntrustedFlowSource::Range {
-  UntrustedSource() { this = any(UntrustedFunction f).getACall() }
-}
-
-class SinkFunction extends Function {
-  SinkFunction() { this.getName() = ["sinkString", "sinkBytes"] }
-}
-
-class TestConfig extends TaintTracking::Configuration {
-  TestConfig() { this = "testconfig" }
-
-  override predicate isSource(DataFlow::Node source) { source instanceof UntrustedFlowSource }
+module Config implements DataFlow::ConfigSig {
+  predicate isSource(DataFlow::Node source) {
+    exists(Function fn | fn.hasQualifiedName(_, ["getUntrustedString", "getUntrustedBytes"]) |
+      source = fn.getACall().getResult()
+    )
```

Comment:
This previously ranged over all of `UntrustedFlowSource`, but is now more limited. Do we miss out on anything as a result of this change?

Reply:
As above, I don't think it matters for tests.